src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs

```diff
@@ -8,34 +8,32 @@ namespace Files.Uwp.UserControls
 {
     public sealed partial class PaneControl : UserControl, IPane
     {
-        public PanePositions Position { get; private set; } = PanePositions.None;
-
-        private IPaneSettingsService PaneService { get; } = Ioc.Default.GetService<IPaneSettingsService>();
```

Comment:
We want to align with this format of defining services 🙂

Reply:
I don't understand your comment. Could you explain?

Reply:
Can you keep the old `PaneService`? We want to avoid `_paneService` naming.

Reply:
So I have to use a private field (not a property) with the name PaneService?

Reply:
No, just use
```csharp
private IPaneSettingsService PaneService { get; } = Ioc.Default.GetService<IPaneSettingsService>();
```

Reply:
It's done.